LayoutTests/imported/w3c/web-platform-tests/html-aam/roles-contextual.html

```diff
@@ -40,6 +40,9 @@
   <header data-testname="el-header" aria-label="x" class="ex-generic">x</header>
 </nav>
 <header data-testname="el-header-ancestorbody" data-expectedrole="banner" class="ex">x</header>
+<main>
+  <header data-testname="el-header-ancestormain" data-expectedrole="generic" class="ex">x</header>
+</main>
```

Comment:
You may be aware, but by adding a testcase to this WPT, we'll need to make sure to export it. The `Tools/Scripts/export-w3c-test-changes` should be able to help.

Reply:
https://github.com/web-platform-tests/wpt/pull/44080